### DISCUSSION TITLE
mcpbridges只能在higress-system命名空间中使用的问题

### DISCUSSION THREAD

**Discussion by w7team** [higress-group/higress#1413] [Q&A]
问题1：mcpbridges只能在higress-system命名空间中创建才能生效，其他命名空间中无效
问题2：ingress也只能跟mcpbridges在同一个命名空间下才能在backend中使用，导致ingress也只能在higress-system下创建

目前是用的k3s测试环境，通过helm安装的higress。现在想把mcpbridges也按命名空间隔离，不然不好做多组户相关的规划。

**Comment by CH3CHO** [comment]
请 @johnlanni 看一下

**Comment by johnlanni** [comment]
ingress 其实可以不跟 mcpbridge 在一个命名空间下的

**Comment by w7team** [comment]
Ingress不在一个命名空间下就是问题2，测试backend不生效，你们测测





------------------ 原始邮件 ------------------
发件人: 澄潭 ***@***.***&gt;
发送时间: 2024年10月17日 11:10
收件人: alibaba/higress ***@***.***&gt;
抄送: 微擎团队 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [alibaba/higress] mcpbridges只能在higress-system命名空间中使用的问题 (Issue #1400)





 
ingress 其实可以不跟 mcpbridge 在一个命名空间下的

**Comment by johnlanni** [comment]
试过，可以的

**Comment by CH3CHO** [comment]
可以检查一下安装参数，有没有指定 Higress 只监听某个 NS 下面的配置，或者 Ingress 配置的 IngressClass 和 Higress 监听的 IngressClass 不一致。

**Comment by w7team** [comment]
好的 我们这边再查查
